twtr/grails-app/assets/javascripts/app/templates/search.tpl.htm

```diff
@@ -30,10 +30,10 @@
             </tr>
             </thead>
             <tbody>
-            <tr ng-repeat="tweet in account">
-                <td ng-click="goToUserDetails()"><a href=""> {{ tweet.handle }}</a></td>
+            <tr ng-repeat="tweet in account | orderBy:dateCreated:reverse ">
```

Comment:
shouldn't this sorting be done in the MessageController.groovy portion of the api?  Maybe it wasn't part of the original requirements.